### PR TITLE
prune the current ownership file

### PR DESCRIPTION
**Pull request by animeshsingh**
before we add new owners

### CODE REVIEW

**Comment by jlewi** [comment]
Thanks!
/lgtm
/approve

/hold

@joeliedtke Are you still actively working on the KF docs?

**Comment by joeliedtke** [comment]
Yes.  I work on the KFP docs.

File: OWNERS [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
   - rmgogogo
-  - janeman98
 reviewers:
   - aronchick
   - carmine
   - dansanche
   - Jeffwan
   - jinchihe
-  - kunmingg
   - lluunn
```
Comment (jlewi) [inline OWNERS:12]: We can remove lluunn

File: OWNERS [hunk 2]
```diff
@@ -27,6 +23,5 @@ reviewers:
   - paveldournov
   - qimingj
```
Comment (jlewi) [inline OWNERS:24]: We can remove @qimingj and @rileyjbauer I think; I believe they have moved onto other projects.

File: OWNERS [hunk 3]
```diff
@@ -1,18 +1,14 @@
...
-  - abhi-g
   - jlewi
-  - richardsliu
   - joeliedtke
   - animeshsingh
   - rmgogogo
-  - janeman98
 reviewers:
   - aronchick
   - carmine
```
Comment (jlewi) [inline OWNERS:8]: We can remove @carmine I think; I don't think he's involved in Kubeflow anymore.

**Pull request merged**